.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -6,32 +6,42 @@ about: Create a report to help us improve
 
 ## Context
 
+<!--
+Check if the issue wasn't already fixed: `npm install --no-save telegraf/telegraf`
+
 Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.
+-->
 
 * Telegraf.js Version:
 * Node.js Version:
 * Operating System:
 
+## Minimal Example Code Reproducing the Issue
+
+```ts
+// your code here. Please omit everything not related to the issue,
+// to make it easier to pinpoint the cause.
+```
+
+<!--
+How to trigger the issue?
+Please provide detailed step-by-step guide.
+-->
+
```

Comment:
Unsure about this, you're asking for a lot here. Providing a code example is nice if we actually have a definite bug or a reproducible crash, but it is not always required. In fact, if you look at the closed issue of the past weeks, most of them could easily be closed without any code, e.g. questions.

Instead of asking less of the authors, we could just as well add another template for questions/other/general/confusion. That is far more common than true bugs, and it could well have its own place. Then we can keep this as focused as it it.

Reply:
I'm thinking about introducing a question template too, but I'm not yet sure how it should differ from the bug report template, so I was planning to do it in next pull request. Advice or help welcome.

I heard there's a way to link issue template to open a discussion instead, but I haven't looked into it yet.

> if you look at the closed issue of the past weeks, most of them could easily be closed without any code

True. On the flip side, we have several open (#1182, #1223) and a bunch of closed issues that are / were difficult to move forward because they have too little or way too much code.

I added `needs more info` label for those, btw. And we could have a bot to auto-close them if inactive.

We could also add "delete anything that's not applicable" comment, and "laziness will make it more difficult for us to help".

Reply:
> Advice or help welcome.

I'll figure something out.

> I added `needs more info` label for those

Nice :+1:

> we could have a bot to auto-close them if inactive

I don't think that's necessary for the scale of our project (unless you want to play around with GH bots, then feel free).

> We could also add "delete anything that's not applicable" comment

That sounds good!

> "laziness will make it more difficult for us to help".

That's rude, don't insult people before you talk to them. A friendly reminder could certainly help.

Reply:
> I heard there's a way to link issue template to open a discussion instead, but I haven't looked into it yet.

Confer https://github.com/eamodio/vscode-gitlens/issues/new/choose